index.html

```diff
@@ -0,0 +1,90 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Random Background Color</title>
+  <script>
```

Comment:
[head] They are parallel call. It might be a problem for different cases 
[head] Blocks the parser until the script is ready
[head] Could use async or defer parameters 


[body] The parser won’t be blocked until the end 
[body] All elements will be accessible by script 
[body] Page will be able to use unless the script is not ready[i assume that the features work on js]